### PR TITLE
feat: feedback service - added filters for feedback listing and impro…

### PR DESCRIPTION
**Pull request by akhilmhdh**
# Closes

# Explain the feature/fix

1. Added better filters for feedback listing
2. Modified getFeedbackBy -> getFeedbackById

## Does this PR introduce a breaking change

No

## Screenshots

<details>
<summary>View Screenshots</summary>

</details>

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [x] Does the change have appropriate unit tests?
- [ ] Did tests pass?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [x] Was this feature demo'd and the design review approved?

### CODE REVIEW

File: packages/feedback-service/src/feedbacks/schema.ts [hunk 1]
```diff
@@ -15,13 +15,11 @@ export const FeedbackSchema: Schema = new Schema({
     path: { type: String },
   },
   category: { type: String, enum: ['BUG', 'FEEDBACK'] },
-  createdOn: { type: Date, default: Date.now },
   createdBy: String,
-  updatedOn: { type: Date, default: Date.now },
   updatedBy: String,
-});
+}, { timestamps: { createdAt: 'createdOn', updatedAt: 'updated_On' } });
```
Comment (riginoommen) [inline packages/feedback-service/src/feedbacks/schema.ts:20]:
```suggestion
}, { timestamps: { createdAt: 'createdOn', updatedAt: 'updated_On' } });
```

Type error for updated_On

File: packages/feedback-service/src/e2e/feedback.spec.ts [hunk 2]
```diff
@@ -18,7 +18,9 @@ fragment feedbackType on FeedbackType {
       name
       email
     }
-    createdBy
+    createdBy{
```
Comment (riginoommen) [inline packages/feedback-service/src/e2e/feedback.spec.ts:21]:
```suggestion
    createdBy {
```

File: packages/feedback-service/src/feedbacks/schema.ts (hunk 1 above)
Comment (akhilmhdh) [reply packages/feedback-service/src/feedbacks/schema.ts]: Done

File: packages/feedback-service/src/e2e/feedback.spec.ts (hunk 2 above)
Comment (akhilmhdh) [reply packages/feedback-service/src/e2e/feedback.spec.ts]: Done

**Review by riginoommen — APPROVED** [review]

**Review by deshmukhmayur — APPROVED** [review]

**Pull request merged**